package.json

```diff
@@ -19,8 +19,8 @@
     "private": true,
     "keywords": [
         "ts",
-        "typescript",
-        "ts plugin",
+        "javascript",
+        "plugin",
         "webstorm"
```

Comment:
I suggest adding `typescript` keyword instead of replacing it with `javascript` as vscode manifest [documentation](https://code.visualstudio.com/api/references/extension-manifest) allows up to 5 keywords

Reply:
Let's add `TypeScript Hero` keyword. It is actually better replacement